Comparison Engine/Comparison Engine/Base Classes/BarManager.cs

```diff
@@ -24,11 +24,31 @@ public void AddBar(string barName, string location)
             barList.Add(new Bar(barID, barName, location));
         }
 
+        //Removes the bar from the list of all bars
+        public void RemoveBar(Bar bar)
+        {
+            barList.Remove(bar);
+        }
+
         //Finds the price of a specific drink at a specific bar
-        public float FindDrinkPrice(int barID, int drink)
+        public float FindDrinkPrice(int barID, int drinkID)
         {
             //Return the price of specified drink at this bar
-            return barList[barID].availableDrinks.ContainsKey(drink) ? barList[barID].availableDrinks[drink] : -1; //Should return something else besides -1 if drink not available here
+            if (barList[barID].availableDrinks.TryGetValue(drinkID, out float value))
+            {
+                return value;
+            }
+            else
+            {
+                Console.WriteLine("FindDrinkPrice failed: bar does not have this drink");
+                return -1;
+            }
+        }
```

Comment:
not sure what changed here and still need to talk about how ti could be connected, cuz that Console.WriteLine is really not looking lovely